dp/subsetsum.py

```diff
@@ -0,0 +1,20 @@
+def isSubsetSum (items, length, summ):
+	if summ == 0:
+		return True
+	if length == 0 and summ != 0:
+		return False
+
+	if (items[length - 1] > summ):
+		return isSubsetSum(items, length - 1, summ);
+
+	return isSubsetSum(items, length - 1, summ) or isSubsetSum(items, length - 1, summ - items[length - 1])
```

Comment:
It looks like the code runs in an exponential time. Please try to optimize using Dynamic programming.